frontend/test/ui/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from foxpuppet import FoxPuppet
+
+
+@pytest.fixture
+def foxpuppet(selenium):
+    return FoxPuppet(selenium)
+
+
+@pytest.fixture
+def capabilities(capabilities):
+    capabilities['marionette'] = True
```

Comment:
It's not the default when using pytest-selenium, which doesn't currently use `DesiredCapabilities.FIREFOX`.